### PR TITLE
DOXYGEN: Review headers from 'audio'

### PR DESCRIPTION
**Pull request by b-gent**
This PR has 3 headers in scope of GSoD.
The files are:
- audiostream.h
- mixer.h
- timestamp.h

The rest of the files are only receiving a doxy group.

### CODE REVIEW

**Comment by b-gent** [comment]
@criezy 
Output is here
https://b-gent.github.io/2655/db/d71/group__audio__audiostream.html

Overall not a huge amount to check here, I think.

File: audio/audiostream.h [hunk 1]
```diff
@@ -305,37 +331,45 @@ class SubSeekableAudioStream : public SeekableAudioStream {
 	Timestamp _pos;
 };
 
+/**
+ * A QueuingAudioStream class that allows for queuing audio streams for playback.
+ * @todo A bit more might be needed here
```
Comment (criezy) [inline audio/audiostream.h:336]: I think it is fine like this. Although maybe we can highlight its specificity a bit more by adding `multiple`:
`A QueuingAudioStream class that allows for queuing multiple audio streams for playback.`

File: audio/timestamp.h [hunk 2]
```diff
@@ -28,128 +28,162 @@
...
+	 * This means that two timestamps can compare as equal
+	 * even if they use different frame rates.
 	 */
 	bool operator==(const Timestamp &ts) const;
+	/**
+	 * Check whether two timestamps describe a different moment in time.
+	 */
 	bool operator!=(const Timestamp &ts) const;
+	/**
+	 * Check whether this timestamp describes an earlier moment in time than another timestamp. @todo very unsure if I understand this correctly
```
Comment (criezy) [inline audio/timestamp.h:123]: Yes, that is correct.

**Review by criezy — CHANGES_REQUESTED** [review]
This looks very good.
It is almost ready to merge.

**Comment by b-gent** [comment]
Comments fixed.

**Review by criezy — APPROVED** [review]
That was quick! And it is now good to merge.

**Pull request merged**